app/models/reaction.rb

```diff
@@ -170,6 +170,8 @@ def permissions
   end
 
   def negative_reaction_from_untrusted_user?
+    return if user.any_admin?
```

Comment:
`return if user&.any_admin?`

Will fix the failing spec.

Reply:
Thanks @jacobherrington. I had made the change locally (all specs run fine for me) but for some reason did not push it.